Article sur la première année d'OpenLayer (#129)
dot-layer/blog: content/blog/2020-07-29-openlayer-one-year/index.fr.md

---
title: OpenLayer, 1 an plus tard
author: David Beauchemin

> date: '2020-07-29'

Author: ```suggestion
date: '2020-08-19'
```

slug: podcast
type: post
categories: ["podcast"]
tags: []
description: "Retour sur la première année d'existence d'OpenLayer"
featured: "OpenLayer_YoutubeBanner.jpg"
featuredpath: "img/headers/"
---

Cela fait maintenant 1 an que j’ai lancé OpenLayer, mon projet de podcast vidéo sur l'IA. 32 épisodes plus tard, beaucoup de choses se sont dites derrière le micro. Ce texte débute par une présentation de mon voyage dans cette aventure et je termine avec un survol de certains moments forts des diverses discussions que j’ai eu la chance d’avoir avec les invités.

## Le projet

D’abord, c’est quoi OpenLayer?

> Ça se veut un lieu ou une *myriade* de personnes ayant des profils aux antipodes l’un des autres, mais gravitant autour de cette *bébitte* qu’est l'IA. De l’expert en recherche en apprentissage automatique en passant par l’avocat en propriété intellectuelle jusqu’aux ressources humaines ou la santé. Nous avons eu droit à beaucoup d’informations sur cet univers selon différents acteurs du milieu.

Reviewer: ```suggestion
Ça se veut un lieu ou une *myriade* de personnes ayant des profils aux antipodes les uns des autres, mais gravitant autour de cette *bébitte* qu’est l'IA. De l’expert en recherche en apprentissage automatique en passant par l’avocat en propriété intellectuelle jusqu’aux ressources humaines ou la santé. Nous avons eu droit à beaucoup d’informations sur cet univers selon différents acteurs du milieu.
```

Reviewer: Je pense que la première phrase n'est pas terminée aussi : un lieu où une myriade de personne... (viennent partager..?)

> Ça se veut un lieu ou une *myriade* de personnes viennent partager et ayant des profils aux antipodes les uns des autres, mais gravitant autour de cette *bébitte* qu’est l'IA. De l’expert en recherche en apprentissage automatique en passant par l’avocat en propriété intellectuelle jusqu’aux ressources humaines ou la santé. Nous avons eu droit à beaucoup d’informations sur cet univers selon différents acteurs du milieu.

Reviewer 2: un lieu **où**

> Ça se veut un lieu où une *myriade* de personnes viennent partager et ayant des profils aux antipodes les uns des autres, mais gravitant autour de cette *bébitte* qu’est l'IA. De l’expert en recherche en apprentissage automatique en passant par l’avocat en propriété intellectuelle jusqu’aux ressources humaines ou la santé. Nous avons eu droit à beaucoup d’informations sur cet univers selon différents acteurs du milieu.

Reviewer: Je crois que quelque chose cloche encore ici...

Reviewer: ```suggestion
Ça se veut un lieu où une *myriade* de personnes ayant des profils aux antipodes les uns des autres, mais gravitant autour de cette *bébitte* qu’est l'IA, viennent partager. De l’expert en recherche en apprentissage automatique en passant par l’avocat en propriété intellectuelle jusqu’aux ressources humaines ou la santé. Nous avons eu droit à beaucoup d’informations sur cet univers selon différents acteurs du milieu.
```

> Cette idée d'avoir un podcast vidéo m’est venue il y a environ 2-3 ans en écoutant Mike Ward sous écoute (j’ai écouté l’ensemble des épisodes ou à peu près #BRAG). Je me suis dit, il y a quelque chose d’intéressant à faire avec cela. Par la suite, les choses se sont mises en branle lorsque j’ai quitté mon appartement d’étudiant laid sur l'avenue Myrand. Un mur « blanc » vieillis par le soleil et les années en arrière-plan accompagner du son de mon voisin qui gueule (littéralement) en gammant c’est pas l’arrière-plan le plus excitant mettons.

Reviewer: ```suggestion
Cette idée d'avoir un podcast vidéo m’est venue il y a environ 2-3 ans en écoutant Mike Ward sous écoute (j’ai écouté l’ensemble des épisodes ou à peu près #BRAG). Je me suis dit, il y a quelque chose d’intéressant à faire avec cela. Par la suite, les choses se sont mises en branle lorsque j’ai quitté mon appartement d’étudiant laid sur l'avenue Myrand. Un mur « blanc » vieillis par le soleil et les années en arrière-plan accompagner du son de mon voisin qui gueule (littéralement) en *gammant* c’est pas l’arrière-plan le plus excitant mettons.
```

Reviewer: Changement: *gammant* en emph.

> Cette idée d'avoir un podcast vidéo m’est venue il y a environ 2-3 ans en écoutant Mike Ward sous écoute (j’ai écouté l’ensemble des épisodes ou à peu près #BRAG). Je me suis dit, il y a quelque chose d’intéressant à faire avec cela. Par la suite, les choses se sont mises en branle lorsque j’ai quitté mon appartement d’étudiant laid sur l'avenue Myrand. Un mur « blanc » vieillis par le soleil et les années en arrière-plan accompagner du son de mon voisin qui gueule (littéralement) en *gammant* c’est pas l’arrière-plan le plus excitant mettons.

Reviewer 2: accompagné

Reviewer 2: gammant: le mot le plus horrible du post !

Reviewer 2: viellit

Reviewer 2: excitant **,** mettons.

Author: non gamming, je sais pas trop comment l'écrire autrement hahahahaha.

Author: 31b8114

### Les débuts

> *Fast-forward* à l’été 2019, j’ai débuté en invitant un collègue et ami, [Nicolas Garneau](https://youtu.be/x9Zo-F0PWQE), pour commencer avec un invité plus « facile » à interagir avec, puisque je le connais bien. C’est un épisode intéressant puisqu’on aborde différents aspects personnels sur sa vie, mais aussi des aspects plus techniques comme la qualité du code en recherche et ce qui se veut être une introduction à la reproductibilité en science.

Reviewer: ```suggestion
*Fast-forward* à l’été 2019, j’ai débuté en invitant un collègue et ami, [Nicolas Garneau](https://youtu.be/x9Zo-F0PWQE), pour commencer avec un invité plus « facile » à interagir avec, puisque je le connais bien. C’est un épisode intéressant puisqu’on aborde différents aspects personnels de sa vie, mais aussi des aspects plus techniques comme la qualité du code en recherche et ce qui se veut être une introduction à la reproductibilité en science.
```

> *Fast-forward* à l’été 2019, j’ai débuté en invitant un collègue et ami, [Nicolas Garneau](https://youtu.be/x9Zo-F0PWQE), pour commencer avec un invité plus « facile » à interagir avec, puisque je le connais bien. C’est un épisode intéressant puisqu’on aborde différents aspects personnels sur sa vie, mais aussi des aspects plus techniques comme la qualité du code en recherche et ce qui se veut être une introduction à la reproductibilité en science.

Reviewer: ```suggestion
*Fast-forward* à l’été 2019, j’ai débuté en invitant un collègue et ami, [Nicolas Garneau](https://youtu.be/x9Zo-F0PWQE), pour commencer avec un invité plus « facile » à interagir avec, puisque je le connais bien. C’est un épisode intéressant puisqu’on aborde différents aspects personnels sur sa vie, mais aussi des sujets plus techniques comme la qualité du code en recherche et ce qui se veut être une introduction à la reproductibilité en science.
```

> *Fast-forward* à l’été 2019, j’ai débuté en invitant un collègue et ami, [Nicolas Garneau](https://youtu.be/x9Zo-F0PWQE), pour commencer avec un invité plus « facile » à interagir avec, puisque je le connais bien. C’est un épisode intéressant puisqu’on aborde différents aspects personnels de sa vie, mais aussi de sujets plus techniques comme la qualité du code en recherche et ce qui se veut être une introduction à la reproductibilité en science.

Reviewer 2: interagir avec : fin d'une clause par une préposition = laid

### L’évolution

> S’en ait suivie plusieurs expérimentations dans les épisodes suivants : le décor à utiliser, la configuration de la console de son, positionnement des micros et de la caméra, le contrôle de la lumière dans l’environnement et de mes chats dans l’arrière-plan. Plusieurs points ont été corrigés, mais d’autres restent à améliorer continuellement.

Reviewer: Je sais pas trop comment gérer le "S'en ait suivie", mais il semble (à ma surprise) que ce ne soit pas accepté comme façon de parler/écrire : https://plus.lesoir.be/128456/article/2017-12-08/sensuivre-ou-sen-suivre

Author: Intéressant.

> Les épisodes suivant m'ont permis d'effectuer plusieurs expérimentations: le décor à utiliser, la configuration de la console de son, le positionnement des micros et de la caméra, le contrôle de la lumière dans l’environnement et de mes chats dans l’arrière-plan. Plusieurs points ont été corrigés, mais d’autres restent à améliorer.

Reviewer: Je ne suis pas certain, mais je crois fortement que suivant prend un "s" ici. Il est l'adjectif d'épisodes, et non pas un participe présent.

Reviewer: http://bdl.oqlf.gouv.qc.ca/bdl/gabarit_bdl.asp?id=2915

Author: Effectivement.

> S’ensuivit plusieurs expérimentations dans les épisodes suivants : le décor à utiliser, la configuration de la console de son, positionnement des micros et de la caméra, le contrôle de la lumière dans l’environnement et de mes chats dans l’arrière-plan. Plusieurs points ont été corrigés, mais d’autres restent à améliorer continuellement.

Reviewer 2: le positionnement

> S’ensuivit plusieurs expérimentations dans les épisodes suivants : le décor à utiliser, la configuration de la console de son, positionnement des micros et de la caméra, le contrôle de la lumière dans l’environnement et de mes chats dans l’arrière-plan. Plusieurs points ont été corrigés, mais d’autres restent à améliorer continuellement.

Reviewer 2: Je n'aime pas s'ensuivit. Premièrement, s'ensuivrent c'est plus approprié. Ensuite, tu as un style familier tout au long du post, ça semble bizzare ce formalisme spontané. Ok si c'est pour capter l'attention, mais tu ne dis rien de plus pertinent que le reste.

Reviewer 2: Il n'y a jamais trop de chats

Reviewer 2: continuellement = mot inutile dans ta phrase.

Author: > Je n'aime pas s'ensuivit. Premièrement, s'ensuivrent c'est plus approprié. Ensuite, tu as un style familier tout au long du post, ça semble bizzare ce formalisme spontané. Ok si c'est pour capter l'attention, mais tu ne dis rien de plus pertinent que le reste.

T'a raison. je vais reformulé.

Author: 53a7b07

### Et maintenant?

On m’a souvent demandé : penses-tu qu’il va y avoir une fin?

Je vais t’avouer que je n’y pense pas vraiment, si ce jour arrive, je vais voir ce que je fais, mais j’ai encore une liste d'au moins 30 invités intéressés à participer à un enregistrement, je suis donc sûrement encore bon pour deux années d’épisodes à ce rythme.

## Les discussions

> À travers les divers épisodes, plusieurs sujets ont été abordés. Notamment l’aspect technique de l’IA ou de l’apprentissage automatique, le droit et la législation des algorithmes et leurs propriétés intellectuelles, la mise en production des solutions et j’en passe. Je pourrais facilement faire un texte avec chaque épisode, c’est pourquoi je vais plutôt recenser ici quelques moments forts qui m’ont personnellement marqué et qui ont amené à avoir une réflexion sur des aspects de mon travail.

Reviewer: ```suggestion
À travers les divers épisodes, plusieurs sujets ont été abordés. Notamment l’aspect technique de l’IA ou de l’apprentissage automatique, le droit et la législation des algorithmes et leurs propriétés intellectuelles, la mise en production des solutions et j’en passe. Je pourrais facilement faire un texte avec chaque épisode, c’est pourquoi je vais plutôt recenser ici quelques moments forts qui m’ont personnellement marqué et qui m'ont amené à avoir une réflexion sur des aspects de mon travail.
```

> À travers les divers épisodes, plusieurs sujets ont été abordés. Notamment l’aspect technique de l’IA ou de l’apprentissage automatique, le droit et la législation des algorithmes et leurs propriétés intellectuelles, la mise en production des solutions et j’en passe. Je pourrais facilement faire un texte avec chaque épisode, c’est pourquoi je vais plutôt recenser ici quelques moments forts qui m’ont personnellement marqué et qui ont amené à avoir une réflexion sur des aspects de mon travail.

Reviewer: ```suggestion
À travers les divers épisodes, plusieurs sujets ont été abordés. Notamment l’aspect technique de l’IA ou de l’apprentissage automatique, le droit et la législation des algorithmes et leurs propriétés intellectuelles, la mise en production des solutions et j’en passe. Je pourrais facilement faire un texte avec chaque épisode, c’est pourquoi je vais plutôt recenser ici quelques moments forts qui m’ont personnellement marqué et qui ont amené à réfléchir sur certains aspects de mon travail.
```
Vraiment juste une suggestion làlà..

> À travers les divers épisodes, plusieurs sujets ont été abordés. Notamment l’aspect technique de l’IA ou de l’apprentissage automatique, le droit et la législation des algorithmes et leurs propriétés intellectuelles, la mise en production des solutions et j’en passe. Je pourrais facilement faire un texte avec chaque épisode, c’est pourquoi je vais plutôt recenser ici quelques moments forts qui m’ont personnellement marqué et qui ont amené à réfléchir sur certains aspects de mon travail.

Reviewer 2: La phrase d'intro au paragraphe est weak considérant que c'est le premier paragraphe de la partie la plus intéressante de ton post

> À travers les divers épisodes, plusieurs sujets ont été abordés. Notamment l’aspect technique de l’IA ou de l’apprentissage automatique, le droit et la législation des algorithmes et leurs propriétés intellectuelles, la mise en production des solutions et j’en passe. Je pourrais facilement faire un texte avec chaque épisode, c’est pourquoi je vais plutôt recenser ici quelques moments forts qui m’ont personnellement marqué et qui ont amené à réfléchir sur certains aspects de mon travail.

Reviewer 2: qui **m'**ont ?

> À travers les divers épisodes, plusieurs sujets ont été abordés avec les invités. Notamment l’aspect technique de l’IA ou de l’apprentissage automatique, le droit et la législation des algorithmes et leurs propriétés intellectuelles, la mise en production des solutions et j’en passe. Je pourrais facilement faire un texte avec chaque épisode, c’est pourquoi je vais plutôt recenser ici quelques moments forts qui m’ont personnellement marqué et qui ont amené à réfléchir sur certains aspects de mon travail.

Reviewer: ```suggestion
À travers les divers épisodes, plusieurs sujets ont été abordés avec les invités. Notamment l’aspect technique de l’IA ou de l’apprentissage automatique, le droit et la législation des algorithmes et leurs propriétés intellectuelles, la mise en production des solutions et j’en passe. Je pourrais facilement faire un texte avec chaque épisode, c’est pourquoi je vais plutôt recenser ici quelques moments forts qui m’ont personnellement marqué et qui m'ont amené à réfléchir sur certains aspects de mon travail.
```

Reviewer: J'y repense, peut-être plus : qui m'ont personnellement marqué et amené à...

Author: Je pense qu'il faut répété avec le m'.

> Premièrement, en réfléchissant au contenu des épisodes, j’en suis venu à un constat; le plus important n’est pas la technologie, ton expertise ou *whatever*, mais principalement la vision que tu portes sur le problème et de comment tu l’utilises pour adresser le problème.

Reviewer 2: ```suggestion
Premièrement, en réfléchissant au contenu des épisodes, j’en suis venu à un constat; le plus important n’est pas la technologie, ton expertise ou *whatever*, mais principalement la vision que tu portes sur le problème et comment tu l’utilises pour adresser le problème.
```

> Prenons par exemple, [Alexandre Dubé-côté](https://youtu.be/YAnDc-GicpY), il a vraiment utilisé une vision orientée affaires pour nous expliquer comment bien concevoir des solutions d'IA. C’est-à-dire, que cette dernière doit répondre à un besoin, qu'il soit un besoin d’affaires, ou académique, ou social, cette solution doit répondre à un besoin.

Reviewer: ```suggestion
Prenons par exemple, [Alexandre Dubé-côté](https://youtu.be/YAnDc-GicpY), il a vraiment utilisé une vision orientée affaires pour nous expliquer comment bien concevoir des solutions d'IA. C’est-à-dire, que cette dernière doit répondre à un besoin, qu'il soit un besoin d’affaires, académique, ou social, cette solution doit répondre à un besoin.
```

> Prenons par exemple, [Alexandre Dubé-côté](https://youtu.be/YAnDc-GicpY), il a vraiment utilisé une vision orientée affaires pour nous expliquer comment bien concevoir des solutions d'IA. C’est-à-dire, que cette dernière doit répondre à un besoin, qu'il soit un besoin d’affaires, académique, ou social, cette solution doit répondre à un besoin.

Reviewer 2: ```suggestion
Prenons par exemple, [Alexandre Dubé-Côté](https://youtu.be/YAnDc-GicpY), qui a vraiment utilisé une vision orientée affaires pour nous expliquer comment bien concevoir des solutions d'IA. Cette dernière doit répondre à un besoin, qu'il soit d’affaires, académique, ou social, sa solution doit répondre à un besoin.
```

> [Helene-Sarah Becotte](https://youtu.be/yRiEF7_i13Q) a beaucoup eu ce discours à travers son utilisation des mathématiques. La vision de beaucoup de gens (dans mon entourage non universitaire, mettons) des maths c’est que c’est *useless*. Alors que si on lui donne une vision plus utilitaire, donc répondre à un besoin, bien les maths prennent tous leurs sens. Le meilleur exemple de cela est son excellent [blog](https://helenebecotte.com/2019/11/24/optimise-ta-productivite-en-remplacant-ton-cafe-par-du-the/) sur l’optimisation de ta consommation de café pour être le plus productif possible.

Reviewer: ```suggestion
[Helene-Sarah Becotte](https://youtu.be/yRiEF7_i13Q) a beaucoup eu ce discours à travers son utilisation des mathématiques. La vision de beaucoup de gens (dans mon entourage non universitaire, mettons) des maths c’est que c’est *useless*. Alors que si on lui donne une vision plus utilitaire, donc répondre à un besoin, bien les maths prennent tout leur sens. Le meilleur exemple de cela est son excellent [blog](https://helenebecotte.com/2019/11/24/optimise-ta-productivite-en-remplacant-ton-cafe-par-du-the/) sur l’optimisation de ta consommation de café pour être le plus productif possible.
```

Reviewer: (prennent tout leur sens)

> [Helene-Sarah Becotte](https://youtu.be/yRiEF7_i13Q) a beaucoup eu ce discours à travers son utilisation des mathématiques. La vision de beaucoup de gens (dans mon entourage non universitaire, mettons) des maths c’est que c’est *useless*. Alors que si on lui donne une vision plus utilitaire, donc répondre à un besoin, bien les maths prennent tout leur sens. Le meilleur exemple de cela est son excellent [blog](https://helenebecotte.com/2019/11/24/optimise-ta-productivite-en-remplacant-ton-cafe-par-du-the/) sur l’optimisation de ta consommation de café pour être le plus productif possible.

Reviewer 2: a beaucoup eu ce discours : weak

Author: dd898a6

> Deuxièmement, sois bon dans au moins deux domaines, je crois que c’est [Gabrielle Trudeau](https://youtu.be/xN7Z8jjxRDU) qui cite son paternel lui ayant dit cette phrase : dans la vie tu dois avoir deux expertises (je n’ai pas réussi à retrouver le moment exact). J’ai trouvé cela fascinant comme concept d’avoir au moins deux expertises et ça m’a fait penser à cette discussion avec [Charles Demontigny](https://youtu.be/0TcH7XWLkIY) qui dit sensiblement la même chose : qu’il aime apprendre 80% de l’essentiel sur plusieurs domaines plutôt que de connaître un seul domaine à 100%.

Reviewer 2: 80/20 = principe de Pareto, pour te ramener à tes racines actuarielles :smiley:

Reviewer 2: J’ai trouvé cela fascinant comme concept d’avoir au moins deux expertises : tu répète ce que tu viens de dire dans la dernière phrase

> J’ai trouvé intéressant l'idée de connaître relativement bien plusieurs domaines, particulièrement de nos jours où la multidisciplinarité est partout. De connaître au minimum le vocabulaire et les différentes notions de base d’un secteur permet de faciliter la communication. Particulièrement lorsque l’on sait que l'IA s'applique à plusieurs domaines (génie, santé, finance ...).

Reviewer 2: la multidisciplinarité est partout : elle est où il y a plusieurs disciplines ? La phrase ne fait pas de sens.

> J’ai trouvé intéressant l'idée de connaître relativement bien plusieurs domaines, particulièrement de nos jours où la multidisciplinarité est partout. De connaître au minimum le vocabulaire et les notions de base d’un secteur, particulièrement lorsque l’on fait de l'IA qui peut s’appliquer sur l’ensemble de ses domaines, me semble important puisque cela permet de faciliter la communication.

Reviewer: l'ensemble de **ces** domaines? Je ne suis pas certain de comprendre.

Author: effectivement. je vais réécrire.

> Ce qui m’amène à mon troisième point, *do the extra miles*! À travers les diverses discussions, j’ai vraiment ressenti que c’est vraiment ce qui fait la différence. Par exemple, se construire un porte-folio de solution, développer son image professionnelle sur les réseaux sociaux (LinkedIn, Twitter, Tik Tok?!?!). Le point est, fait en un petit peu plus que les autres et ça va paraître sur le long run.

Reviewer 2: j’ai vraiment ressenti que c’est vraiment ce qui fait la différence: weak

Reviewer 2: _long run_

> Écoute, je pourrais encore continuer, mais l’idée reste de faire un retour sur cette expérience plus qu’analyser chaque moment. J’espère continuer cette aventure encore longtemps et merci à tout mes ***number one fan*** (au singulier parce que vous êtes unique).

Reviewer 2: tous

Author: Non c'est l'idée d'avoir une faute.

Reviewer 2: mais ça fit pas avec mes. Tu peux dire que tu as plusieurs number one fan mais pas un plusieurs number one fan

Author: Je comprend. je change.

> Écoute, je pourrais encore continuer, mais l’idée reste de faire un retour sur cette expérience plus qu’analyser chaque moment. J’espère continuer cette aventure encore longtemps et merci à tout mes ***number one fan*** (au singulier parce que vous êtes tous unique).

Reviewer: ```suggestion
Écoute, je pourrais encore continuer, mais l’idée reste de faire un retour sur cette expérience plus qu’analyser chaque moment. J’espère continuer cette aventure encore longtemps et merci à tout mes ***number one fan*** (au singulier parce que vous êtes tous uniques).
```
